policyengine_canada/variables/gov/provinces/ns/tax/income/credits/nsprc/ns_poverty_reduction_credit.py

```diff
@@ -0,0 +1,20 @@
+from policyengine_canada.model_api import *
+
+
+class ns_poverty_reduction_credit(Variable):
+    value_type = float
+    entity = Household
+    label = "Nova Scotia Poverty Reduction Credit"
+    unit = CAD
+    definition_period = YEAR
+    reference = (
+        "https://novascotia.ca/coms/noteworthy/PovertyReductionCredit.html"
```

Comment:
```suggestion
        "https://novascotia.ca/coms/noteworthy/PovertyReductionCredit.html",
        "https://nslegislature.ca/sites/default/files/legc/statutes/income%20tax.pdf#page=121"
```

This legislative reference doesn't define the program but does assign responsibility for defining it to the governor.